src/app/static/src/tests/root/actions.test.ts

```diff
@@ -319,12 +320,14 @@ describe("root actions", () => {
 
         const expectedError = {error: "OTHER_ERROR", detail: "TestError"};
         expect(commit.mock.calls[0][0]).toEqual({payload: expectedError, type: RootMutation.ErrorReportError});
+        expect(dispatch.mock.calls.length).toBe(0)
         expect(mockAxios.history.post.length).toEqual(1)
         expect(mockAxios.history.post[0].url).toEqual(url)
 
         const expected = {
             email: "some.user@example.com",
             country: "",
+            projectName: "",
```

Comment:
Since this object is called "expected" this should be "no associated project", country and jobId should be set similarly and then below assert `expect(data.country).toBe(expected.country)` etc.